tools/autograd/gen_variable_type.py

```diff
@@ -344,7 +340,6 @@
 PROFILE_DISPATCH_UNBOXED = CodeTemplate("""\
 static auto op = c10::Dispatcher::singleton().findSchema({"aten::${operator_name}", "${overload_name}"});
 TORCH_INTERNAL_ASSERT(op);
-RECORD_FUNCTION("${name}", std::vector<c10::IValue>({${input_names}}), Node::peek_at_next_sequence_nr());
 return c10::Dispatcher::singleton().callUnboxedRedispatch<${ret_and_arg_types}>(${profiled_dispatch_args});
 """)
```

Comment:
Wrappers should probably be deleted if we're entirely lifting `RECORD_FUNCTION` into the dispatcher

Reply:
yes, I will remove them later, we have some code that uses them (but they become functionally no-op)